meta-resin-common/recipes-core/resin-rollback/files/rollback-altboot.service

```diff
@@ -0,0 +1,28 @@
+# Copyright 2018 Resinio Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Run rollback-altboot.sh to run rollback checks
+DefaultDependencies=no
+Requires=resin-boot.service mnt-sysroot-active.service mnt-sysroot-inactive.service
+Before=balena-host.service balena.service
+After=mnt-sysroot-active.service mnt-sysroot-inactive.service resin-boot.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/bin/sh /usr/bin/rollback-altboot.sh
```

Comment:
if a file goes in `/usr/bin` the resinOS convention is to have it be executable and also not have the `.sh` extension. See the `update-resin-supervisor` script for example